### ISSUE TITLE
using .sublime-syntax

### ISSUE THREAD

**Issue by jesselucas** [jmacdonald/amp#76]
Loving amp and excited to discover all of it's usefulness! 

I've added a new `.sublime-syntax` to the `syntaxes` folder and attempting to use it for bash scripts. It's a syntax file from Sublime 3 called `Shell-Unix-Generic.sublime-syntax' but it doesn't seem to highlight.

Do I need to set it in my preferences? Any guidance would be greatly appreciated.

**Comment by jmacdonald** [comment]
Hey @jesselucas, glad to hear you like it! 🙂 

The `.sublime-syntax` file can specify applicable extensions, which is how Amp resolves the correct syntax. It may just be a matter of appending a `.sh` extension to the file, or if you've already done that, perhaps adding `sh` to the list of extensions in your `.sublime-syntax` file.

Thinking aloud, but the `syntect` library Amp leverages to handle loading/linking syntaxes supports format detection from the first line of the file, although admittedly this is the first scenario for which I'm realizing that would be useful. I suppose Amp should eventually allow you to set or override the syntax for the current buffer, too. I can investigate these for a future release, but do let me know if the aforementioned extension-based workaround will unblock you in the interim!

**Comment by jesselucas** [comment]
Thank you for the reply!

The built in highlighter seems to be working great for `.sh` files. My specific use case is to have syntax highlighting for a `.bashrc` file. The `.sublime-syntax` file I'm trying has `.bashrc` as a file_extension but I still can't get it to work. I feel like I'm missing something simple.

Here's a gist to the `.sublime-syntax` https://gist.github.com/jesselucas/5f70a27304077537fcf05d71a2aeafa4

**Comment by jmacdonald** [comment]
@jesselucas try removing the `.` prefix from the `.bashrc` file extension in that syntax. `scribe` strips anything before (and including) the final `.` when determining the file extension; in this case, it's viewing the extension as `bashrc`. :slightly_smiling_face:

**Comment by jedahan** [comment]
It would be great if people shared good working syntaxes and themes

https://github.com/jmacdonald/amp/wiki

**Comment by jesselucas** [comment]
Thank you @jmacdonald ! That fixed the problem 👏 🎉
